app/services/variant_units/unit_prices.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module VariantUnits
+  class UnitPrices
```

Comment:
I'd rather name it singular, as a [value object](https://www.sitepoint.com/value-objects-explained-with-ruby/). The plural name makes me wonder about its actual intent.